### ISSUE TITLE
Anysoft Keyboard Editing

### ISSUE THREAD

**Issue by ChaoticHub** [AnySoftKeyboard/AnySoftKeyboard#1806]
Hello,

I really wanted to see if it was possible to edit keyboards to remove or add specific keys? I love the Latin keyboard with symbols but having the ' there messes up the entire keyboard spacing. I really want to find some way to mimic the SwiftKey keyboard in terms of layout and symbols in the back of the keys like this.
![signal-attachment-2019-08-04-175014](https://user-images.githubusercontent.com/35610319/62429735-5e262b00-b6e0-11e9-9ac1-35c37c77a6f9.jpeg)

Also, is it possible to remove the last previous word whenever holding down backspace? Is there a tweak that I may not be noticing?

Even if this were an option that I would have to pay for it would be worth it, especially since Microsoft now owns SwiftKey so it's kind of ruined with data collection.

Any help would be greatly appreciated

**Comment by IronJunkie** [comment]
I would also prefer this key wasn't there and asdfg row was half key offset like most other keyboards.

You could edit your own layout, if you can code 😬, though better option might be if that key could be disabled by a setting in "tweaks" menu

shift+backspace will delete last word (whole words), is that what you mean?

**Comment by jwsp1** [comment]
Swiftkey is probably one of the most used android keyboards and its design is super awesome. Hope that someone will create a layout like this...

**Comment by adeyo50** [comment]
I agree. Swiftkey is the worlds most popular key for a reason. My favorite feature are punctuations -simply a swype on the punctuation key towards the spacebar. I've yet to find anything as easy or fast. I'd also like to get the microphone working with a FOSS dictation service. I installed a foss voice typing engine on my galaxy note 20 but found that something prevented it from using my default -instead, anysoft keyboard kept using googles which was no longer set as my default. :(
